### PR TITLE
Bump crossplane/crossplane-runtime version from v0.10.0 to v0.13.0

### LINKED ISSUE #1295: Bump crossplane/crossplane-runtime from alpha version to official version
**Is your feature request related to a problem? Please describe.**

In version v0.13.0, the API of crossplane/crossplane-runtimehas has reached its official version `v1`. Considering bump api `github.com/crossplane/crossplane-runtime/apis/core/v1alpha1` v0.10.0 to `github.com/crossplane/crossplane-runtime/apis/common/v1` v0.13.0.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by zzxwill**
In version v0.13.0, the API of crossplane/crossplane-runtime has
reached its official version v1. Considering bump api
github.com/crossplane/crossplane-runtime/apis/core/v1alpha1
v0.10.0 to github.com/crossplane/crossplane-runtime/apis/common/v1 v0.13.0.

Fix #1295

### CODE REVIEW

File: pkg/controller/core.oam.dev/v1alpha2/applicationconfiguration/component.go [hunk 1]
```diff
@@ -53,7 +53,8 @@ type ComponentHandler struct {
 
 // Create implements EventHandler
 func (c *ComponentHandler) Create(evt event.CreateEvent, q workqueue.RateLimitingInterface) {
-	reqs, succeed := c.createControllerRevision(evt.Meta, evt.Object)
+	// TODO(zzxwill) evt.Meta in controller-runtime v0.6.2 is gone
+	reqs, succeed := c.createControllerRevision(evt.Object, evt.Object)
```
Comment (zzxwill) [inline pkg/controller/core.oam.dev/v1alpha2/applicationconfiguration/component.go:57]: @wonderflow @ryanzhang-oss In Kubernetes controller-runtime v0.8.0, event.CreateEvent doesn't have field `Meta`. I haven't found a way to fix it. Please help take look at it.

File: cmd/core/main.go [hunk 2]
```diff
@@ -300,21 +277,3 @@ func waitWebhookSecretVolume(certDir string, timeout, interval time.Duration) er
 		}
 	}
 }
-
-func makeSignalHandler() (stopCh <-chan struct{}) {
```
Comment (wonderflow) [inline cmd/core/main.go:304]: why delete it?

File: go.mod [hunk 3]
```diff
@@ -61,33 +61,30 @@ require (
...
-	k8s.io/api v0.18.8
-	k8s.io/apiextensions-apiserver v0.18.6
-	k8s.io/apimachinery v0.18.8
-	k8s.io/cli-runtime v0.18.6
-	k8s.io/client-go v12.0.0+incompatible
+	k8s.io/api v0.20.2
+	k8s.io/apiextensions-apiserver v0.20.2
+	k8s.io/apimachinery v0.20.2
+	k8s.io/cli-runtime v0.20.2
+	k8s.io/client-go v0.20.2
```
Comment (wonderflow) [inline go.mod:71]: does it necessary to bump k8s api version?

File: go.mod [hunk 4]
```diff
@@ -61,33 +61,30 @@ require (
...
 	k8s.io/klog v1.0.0
 	k8s.io/klog/v2 v2.4.0
-	k8s.io/kube-openapi v0.0.0-20200410145947-bcb3869e6f29
-	k8s.io/kubectl v0.18.6
+	k8s.io/kube-openapi v0.0.0-20201113171705-d219536bb9fd
+	k8s.io/kubectl v0.20.2
 	k8s.io/utils v0.0.0-20201110183641-67b214c5f920
-	sigs.k8s.io/controller-runtime v0.6.2
+	sigs.k8s.io/controller-runtime v0.8.0
```
Comment (wonderflow) [inline go.mod:77]: does it necessary?

**Pull request closed without merging**